pandas/core/indexes/base.py

```diff
@@ -3632,8 +3632,7 @@ def values(self):
         return self._data.view(np.ndarray)
 
     @property
-    def _values(self):
-        # type: () -> Union[ExtensionArray, Index, np.ndarray]
+    def _values(self) -> Union[ExtensionArray, 'Index', np.ndarray]:
```

Comment:
Is Index not a valid type here?

Reply:
Because it's used inside the Index class itself, `Index` isn't defined yet when types are checked, so we need to use a forward reference literal. See [PEP 484](https://www.python.org/dev/peps/pep-0484/#forward-references). In that context, `Index` alone is an unresolved reference.